package.json

```diff
@@ -15,69 +15,75 @@
         "type": "git",
         "url": "https://github.com/ekino/veggies.git"
     },
-    "main": "src/index.js",
-    "bin": "./bin/veggies.js",
```

Comment:
i would let this in place

Reply:
The bin is always there. I just changed the format of bin for more clear:
```json
    "bin": {
        "veggies": "./bin/veggies.js"
    },

```

Reply:
Sorry, i didn't see. Thanks